config/maximal-config-example.toml

```diff
@@ -59,14 +59,15 @@ syncing_interval_secs = 60
 recently_closed_allocation_buffer_secs = 3600
 
 [subgraphs.escrow]
+# NOTE: It is heavily recomended to use both `query_url` and `deployment_id`,
+# however it can work with either one.
```

Comment:
`query_url` is mandatory so you can not use either one. It must have `query_url` and it's also recommended to set a `deployment_id`.